webrtc-c/canary/src/Config.cpp

```diff
@@ -0,0 +1,85 @@
+#include "Include.h"
+
+namespace Canary {
+
+STATUS mustenv(CHAR const* pKey, const CHAR** ppValue)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+
+    CHK(ppValue != NULL, STATUS_NULL_ARG);
+
+    CHK_ERR((*ppValue = getenv(pKey)) != NULL, STATUS_INVALID_OPERATION, "%s must be set", pKey);
+
+CleanUp:
+
+    return retStatus;
+}
+
+BOOL getenvBool(CHAR const* pKey)
+{
+    PCHAR pValue;
+
```

Comment:
Really minor: I recommend people using a single return from within a function. For a super simple one like this it's OK. However, if the function is longer and more complex, having a pre-initialized variable to default (say FALSE) and then overriding it on discovery of the param being set is the right thing to do

Reply:
Makes sense. But, since I've modified this to use mustenv, I have to return STATUS instead. Now, I have it as normal pattern with `CHK_*` and `CleanUp`